cbinfoblox/bridge.py

```diff
@@ -53,9 +54,8 @@ def run(self):
                     sensors = self.cb.select(Sensor).where('ip:{}'.format(sensor_ip))
                     if len(sensors) == 0:
                         logger.error("No sensors found with IP: {}".format(sensor_ip))
-                        continue
```

Comment:
I like the idea behind this :+1:. Just make sure that the "actions" will then fail gracefully if there was no sensor found for the given IP address.